src/genie/libs/parser/nxos/show_logging.py

```diff
@@ -74,3 +74,62 @@ def cli(self, include='', output=None):
                 continue
 
         return parsed_dict
+
+class ShowLoggingLevelSchema(MetaParser):
+    '''Schema for:
+        * 'show logging level'
+        * 'show logging level {name}'
+    '''
+
+    schema = { 'facility':
+                { Any():
+                    { 'def_severity': int,
+                      'curr_severity': int }
+                }
+            }
+
+class ShowLoggingLevel(ShowLoggingLevelSchema):
+    '''Schema for:
+        * 'show logging level'
+        * 'show logging level {name}'
+    '''
+
+    cli_command = [ 'show logging level', 'show logging level {name}' ]
+
+    def cli(self, name='', output=None):
```

Comment:
It might be better to have this be `facility=''` to closer reflect the NXOS command